build/package/proxy.Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM alpine:latest
 
-ENV API_PORT 8001
 ENV ENV local
+ENV DATABASE_URL postgres:5432
+ENV DATABASE_USERNAME user
+ENV DATABASE_PASSWORD pass
+ENV DATABASE_TABLE tezoslink?sslmode=disable
```

Comment:
Du coup si j'override DATABASE_TABLE, ca va supprimer le `?sslmode=disable` ?

Reply:
Yes, only meant for local dev